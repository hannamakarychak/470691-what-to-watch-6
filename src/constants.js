export const ALL_GENRES = `All genres`;

export const AuthorizationStatus = {
  AUTH: `AUTH`,
  NO_AUTH: `NO_AUTH`
};
