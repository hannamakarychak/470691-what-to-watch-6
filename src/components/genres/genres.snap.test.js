import React from 'react';
import {render} from '@testing-library/react';

import Genres from './genres';
import {ALL_GENRES} from '../../constants';

it(`Should Genres render correctly`, () => {
  const {container} = render(
    <Genres
      genres={[ALL_GENRES, `Comedy`]}
      selectedGenre="Comedy"
      onGenreSelect={jest.fn}
    />
  );
  expect(container).toMatchSnapshot();
});
