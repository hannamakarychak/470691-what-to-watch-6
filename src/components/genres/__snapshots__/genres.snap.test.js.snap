// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Genres render correctly 1`] = `
<div>
  <ul
    class="catalog__genres-list"
  >
    <li
      class="catalog__genres-item "
    >
      <a
        class="catalog__genres-link"
        href="#"
      >
        All genres
      </a>
    </li>
    <li
      class="catalog__genres-item catalog__genres-item--active"
    >
      <a
        class="catalog__genres-link"
        href="#"
      >
        Comedy
      </a>
    </li>
  </ul>
</div>
`;
